docker/falco/versions.yaml

```diff
@@ -0,0 +1,52 @@
+# Copyright 2022 Google LLC
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy of
```

Comment:
This header doesn't seem to be Apache 2.0 as falco

https://github.com/falcosecurity/falco/blob/master/COPYING

Reply:
Check another Apache container : https://github.com/GoogleCloudPlatform/click-to-deploy/tree/master/docker/solr